boskos/README.md

```diff
@@ -0,0 +1,51 @@
+# WIP boskos
+
+
+## Background
+[βοσκός](https://en.wiktionary.org/wiki/%CE%B2%CE%BF%CF%83%CE%BA%CF%8C%CF%82) - cattle-master in greek!
+
+boskos is a http service that allow k8s test jobs to run independently from GCP projects.
+
+## Introduction
+
+Imagine boskos is a ranch with cattle, and to perform each job in the ranch you need to hold a cattle. 
```

Comment:
Will you change this introduction to explain how it operates without depending on the cattle metaphor?

PS: cattle is like cheese or wheat, you talk about them in bulk rather than as individuals.